### PR TITLE
Paraiba parser rewrite

### PR DESCRIPTION
**Pull request by AnyaLindstromBattle**
Paraiba parser has been completely re-written to be able to parse another dataset which included UUID and date of confirmation.

### CODE REVIEW

File: ingestion/functions/parsing/brazil_paraiba/paraiba.py [hunk 1]
```diff
@@ -12,188 +12,218 @@
...
+    "Gestante": "pregnancy",
+    "Doenças respiratórias crônicas descompensadas": "respiratory system disease",
+    "Doenças renais crônicas em estágio avançado (graus 3, 4 e 5)": "chronic kidney disease",
+    "Doenças cardíacas crônicas": "heart disease",
     "Obesidade": "obesity",
-    "Transtorno mental": "mental or behavioural disorder",
-    "Hipertensão": "hypertension"
+}
+
+_NONE_TYPES = ["Não", "null", "undefined"]
```
Comment (attwad) [inline ingestion/functions/parsing/brazil_paraiba/paraiba.py:41]: use a set(["Não", "null", "undefined"]) here for quick inclusion checks

**Review by attwad — APPROVED** [review]

**Comment by attwad** [comment]
Tests are failing though, please fix

**Comment by AnyaLindstromBattle** [comment]
@Mougk thoughts on symptoms not in the symptom ontology? Namely 'coriza' (line 45). Are these OK to include as symptoms or should I put them in notes as with comorbidities

**Comment by Mougk** [comment]
put in notes.

**Comment by Mougk** [comment]
@attwad can you do rapid review ?

**Review by attwad — APPROVED** [review]

**Pull request merged**